app/proposals_whitelisting.go

```diff
@@ -18,6 +20,19 @@ import (
 	stakeibctypes "github.com/Stride-Labs/stride/v12/x/stakeibc/types"
 )
 
+var WhiteListModule = map[string]struct{}{
```

Comment:
Can you explain this list / where did it come from? Does this restrict proposals to param changes? 

Do we also need to whitelist software upgrade proposals in the gov module?

Reply:
Good question!
That came from here.
https://github.com/cosmos/interchain-security/blob/main/app/consumer-democracy/proposals_whitelisting.go#L41-L52C1

That doesn't restrict proposal change proposals at all.
IsWhiteListModule is only used for whitelisting non-legacy proposals.
For soft-upgrade proposal, if we submit as legacy proposal, we don't need to add software upgrade proposal type to WhitelistModule.
But if we submit as non-legacy proposal, the software upgrade proposal must be whitelisted in WhitelistModule.
To enable both cases, I will add cancel/upgrade proposal types to WhitelistModule.

Reply:
Thanks for adding this. Will follow up with Informal to ask what set of permissions they recommend.